presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -92,7 +92,7 @@
 @ThreadSafe
 public class QueryStateMachine
 {
-    public static final Logger QUERY_STATE_LOG = Logger.get(QueryStateMachine.class);
+    public static final Logger log = Logger.get(QueryStateMachine.class);
```

Comment:
Revert this change. This is not standard class-level logger. It's shared between 2 classes.

Note: I am not arguing this is a good idea to share a logger between classes. However, as long as it's shared, it deserves special name.

Also, you can make it package private at least.